pkg/api/types.go

```diff
@@ -156,8 +158,22 @@ type Config struct {
 
 	// Usage allows for properly shortcircuiting s2i logic when `s2i usage` is invoked
 	Usage bool
+
+	// Injections specifies a list source/destination folders that are injected to
+	// the container that runs assemble. All files are truncated after the
+	// assemble script runs, so they wont be present in the output image.
```

Comment:
@rhcarvalho it is not possible to recover them when the image is comitted. I can add "shred" there before I truncate them, but I think that is an overkill.